src/wallet/redux/sagas/blockchain.ts

```diff
@@ -155,38 +156,47 @@ function handleError(action, err) {
   return put(action(message));
 }
 
-function* watchAdjudicator(deployedContract) {
+function convertBigNumberArgsToBN(argumentObject): any {
```

Comment:
Dealing with different BigNumber libraries is a pain. It'd be nice to find a more maintainable solution.